commands/eliteDangerous.js

```diff
@@ -20,11 +20,19 @@ async function updateFactionStatus(bot) {
     let channelID = "549808289811267602";
     let channel = bot.channels.cache.get(channelID);
     channel.setTopic(topic);
+  } catch (e) { u.errorHandler(e, "Elite Channel Update Error"); }
 
+  let content = "";
+  try {
     // Galnet articles
-    let latestArticle = (await elite.getGalnetFeed())[0];
-    if (latestArticle.title !== lastGalnetArticleTitle) {
-      let content = latestArticle.content.replace(/<br \/>/g, "\n");
+    let latestArticle = await elite.getGalnetFeed().catch(u.noop);
+    if (!latestArticle) { return; }
+    if (latestArticle[0].title !== lastGalnetArticleTitle) {
+      content = latestArticle[0].content.replace(/<br \/>/g, "\n");

```

Comment:
Lines 31 and 36 still refer to `latestArticle`. Setting that variable to the first value in the array would be more streamlined than adding the `[0]` index to every location.
```suggestion
      latestArticle = latestArticle[0];
      content = latestArticle.content.replace(/<br \/>/g, "\n");
```